docs/langref/relay_op.rst

```diff
@@ -35,14 +42,32 @@ This level enables typical convnet models.
    :nosignatures:
 
    tvm.relay.nn.conv2d
+   tvm.relay.nn.max_pool2d
+   tvm.relay.nn.avg_pool2d
+   tvm.relay.nn.global_max_pool2d
+   tvm.relay.nn.global_avg_pool2d
+   tvm.relay.upsampling
```

Comment:
upsampling should belong to relay.image? as it is image operator

Reply:
In NNVM it's under nn.
cc @masahi

Reply:
why not tvm.relay.nn.upsampling?

upsampling is a standard nn op.

Reply:
ok @masahi